.github/scripts/nft_boost/activateBoost.ts

```diff
@@ -0,0 +1,226 @@
+      abi: BackersManagerAbi,
+      functionName: 'rewardToken',
+      args: [],
+    })
+    return getAddress(rewardTokenAddress)
+  }
+
+  // Queries Gauges contracts for backer rewards
+  interface EstimatedGaugeRewards {
+    RBTC: bigint
+    RIF: bigint
+  }
+  const estimatedGaugeRewards = async (backer: Address, gauge: Address): Promise<EstimatedGaugeRewards> => {
+    const rewardTokenAddress = await getRewardTokenAddress()
+    const rewardTokens: Address[] = [rewardCoinbaseAddress, rewardTokenAddress]
+    try {
+      let gaugeEstimatedRewards = await publicClient.multicall({
+        contracts: rewardTokens.map(token => ({
+          address: gauge,
+          abi: GaugeAbi,
+          functionName: 'estimatedBackerRewards',
+          args: [token, backer],
+        })),
+      })
+      gaugeEstimatedRewards = gaugeEstimatedRewards.map(data => data.result)
+      return {
+        RBTC: BigInt(gaugeEstimatedRewards[0]),
+        RIF: BigInt(gaugeEstimatedRewards[1]),
+      }
+    } catch (error) {
+      console.error(`Error fetching rewards for ${backer}:`, error)
+      throw error
+    }
+  }
+
+  return {
+    getLatestBlockNumber,
+    getNftTransferEvents,
+    getAllGauges,
+    estimatedGaugeRewards,
+  }
+}
+
+async function main() {
+  const { nftContractAddress, boostPercentage, env } = args.reduce<Args>((acc, val) => {
+    if (val.startsWith('--nft')) {
+      acc.nftContractAddress = getAddress(val.split('=')[1])
+    }
+    if (val.startsWith('--boost')) {
+      acc.boostPercentage = parseFloat(val.split('=')[1])
+    }
+    if (val.startsWith('--env')) {
+      acc.env = val.split('=')[1]
+    }
+    return acc
+  }, {} as Args)
+
+  if (!env || !nftContractAddress || isNaN(boostPercentage)) {
+    throw new Error(
+      'Usage: npx tsx .github/scripts/nft_boost/activateBoost.ts \\\
+      --nft=<nftContractAddress> \\\
+      --boost=<boostPercentage> \\\
+      --env=<env>',
+    )
+  }
+  console.info('nft: ', nftContractAddress)
+  console.info('Boost percentage: ', boostPercentage)
+  console.info('env: ', env)
+  if (boostPercentage < 0) {
+    throw new Error('Boost percentage must be positive')
+  }
+
+  envConfig({
+    path: `.env.${env}`,
+  })
+
+  const { getLatestBlockNumber, getNftTransferEvents, getAllGauges, estimatedGaugeRewards } =
+    await getActions()
+
+  const blockNumber = await getLatestBlockNumber()
+  console.info(`Latest block: ${blockNumber}`)
+
+  const nftTransferEvents = await getNftTransferEvents(nftContractAddress)
+  const holdersData: any = {}
+
+  const gauges = await getAllGauges()
+
+  const normalisationFactor = 100_000 // Scaling factor to handle decimal values in BigInts. A ratio of 1.25 => 125_000
+  const boost = BigInt(Math.round(normalisationFactor * (1 + boostPercentage / 100)))
+
+  for (let i = 0; i < nftTransferEvents.length; i++) {
+    const event = nftTransferEvents[i]
+    const holderAddress = getAddress(`0x${event.topics[2].slice(-40)}`)
```

Comment:
according to the spec, this address may be the `0x00...00` address when the NFT is destroyed. Should we handle this case?